dir.props

```diff
@@ -123,11 +139,16 @@
 
     <DnuRestoreCommand>"$(DnuToolPath)"</DnuRestoreCommand>
     <DnuRestoreCommand>$(DnuRestoreCommand) restore</DnuRestoreCommand>
+
+    <DnuRestoreCommandForBuildTestsAgainstPackages Condition="'$(BuildTestsAgainstPackages)' == 'true'">$(DnuRestoreCommand) --packages "$(PackageInstallPath)" --source $(PackagesDir)</DnuRestoreCommandForBuildTestsAgainstPackages>
```

Comment:
@karajas It looks like this is restoring the packages from corefx\packages to corefx\bin\localpackages, and then the test runtime will use localpackages as the dependency source because `PackagesDir` is overridden on line 585 of this file.  However, the current operation of BuildTestsAgainstPackages is to go in the opposite direction, restoring the packages from corefx\bin\localpkg to corefx\packages, and then using corefx\packages as the dependency source.  Would it make more sense to continue restoring in that direction and using corefx\packages as the dependency source?

Also, all of the runtime packages would already be in corefx\packages, so I don't think you'd have to explicitly list them out in buildagainstpackagesdeps.props to copy them to bin\localpackages like you're doing now.